panos/network.py

```diff
@@ -1821,6 +1845,45 @@ def _setup(self):
         self._params = tuple(params)
 
 
+class PathMonitorDestination(VersionedPanObject):
+    """PathMonitorDestination Static Route 
+
+    Args:
+        name (str): Name of Path Monitor Destination 
+        enabled (bool): Enable Path Monitor Destination 
```

Comment:
Seems to be `enable` not `enabled` down below, just need to tweak this docstring.